iTjuana/itj-library
# adding script to add books from google sheets

### Description of the Change

This is the first version of script to add books to database from Google Sheets, there are things to improve/refactor, but it gets the job done.

### Verification Process



1. Ask @fabianshady  for the complete address to add it in src/scripts/addBooks.ts

2. Check for any update in the [books Google Sheet](https://docs.google.com/spreadsheets/d/1eg2-pNo_qFHT4wKdyrZiPrNabIlbThPHr5Qn892-ng8/edit?usp=sharing)

3. Run the following command:
`npm run addbooks`

4. The command will show info about the process status of each book


### Applicable Issues


- #95
